reference-implementation/run-tests.js

```diff
@@ -2,12 +2,14 @@ const glob = require('glob');
 const path = require('path');
 
 import ReadableStream from './lib/readable-stream';
+import { IsReadableStreamDisturbed } from './lib/readable-stream';
 import WritableStream from './lib/writable-stream';
 import ByteLengthQueuingStrategy from './lib/byte-length-queuing-strategy';
 import CountQueuingStrategy from './lib/count-queuing-strategy';
 import TransformStream from './lib/transform-stream';
 
 global.ReadableStream = ReadableStream;
+global.IsReadableStreamDisturbed = IsReadableStreamDisturbed;
```

Comment:
Yes, that sounds good. And maybe move them to a separate abstract-ops.js or from-other-specs.js test file.